scripts/test_updates_pg14.sh

```diff
@@ -12,4 +12,6 @@ run_tests "$@" -v7 \
 run_tests "$@" -v8 \
           2.5.0-pg14 2.5.1-pg14 2.5.2-pg14 2.6.0-pg14 2.6.1-pg14 2.7.0-pg14 2.7.1-pg14 2.7.2-pg14 \
           2.8.0-pg14 2.8.1-pg14 2.9.0-pg14 2.9.1-pg14 2.9.2-pg14 2.9.3-pg14 2.10.0-pg14 2.10.1-pg14
+run_tests "$@" -r -v8 \
+          2.10.0-pg14 2.10.1-pg14
```

Comment:
```suggestion
run_tests "$@" -v8 \
          2.5.0-pg14 2.5.1-pg14 2.5.2-pg14 2.6.0-pg14 2.6.1-pg14 2.7.0-pg14 2.7.1-pg14 2.7.2-pg14 \
          2.8.0-pg14 2.8.1-pg14 2.9.0-pg14 2.9.1-pg14 2.9.2-pg14 2.9.3-pg14
run_tests "$@" -r -v8 \
          2.10.0-pg14 2.10.1-pg14
```
Don't need to run twice tests for 2.10.x versions.